kv/range_cache.go

```diff
@@ -162,10 +162,10 @@ func (rmc *rangeDescriptorCache) EvictCachedRangeDescriptor(descKey proto.Key, s
 	}
 
 	for !bytes.Equal(descKey, proto.KeyMin) {
+		rmc.rangeCache.Del(rngKey)
 		if log.V(1) {
-			log.Infof("evict cached descriptor: key=%s desc=%+v\n%s", descKey, cachedDesc, rmc)
+			log.Infof("evicted cached descriptor: key=%s desc=%+v\n", descKey, cachedDesc)
```

Comment:
The `RangeCache.String()` info was very useful for debugging range cache problems in the past. Perhaps add a `RangeCache.StringLocked()` method and use it here.